.github/workflows/ci.yml

```diff
@@ -123,6 +137,7 @@ jobs:
     # This prevents duplicate CI runs for our own pull requests, whilst preserving the ability to
     # run the CI for each branch push to a fork, and for each pull request originating from a fork.
     if: github.event_name == 'push' || github.event.pull_request.head.repo.id != github.event.pull_request.base.repo.id
+    needs: [build, test]
```

Comment:
`package` can run in parallel to `build` and `test` as it does not depend on any of their outputs - it will speed up the global CI time

Reply:
```suggestion
```